modules/configuring-ovnk-additional-networks.adoc

```diff
@@ -6,14 +6,28 @@
 [id="configuration-ovnk-additional-networks_{context}"]
 = Configuration for an OVN-Kubernetes additional network
 
-The {openshift-networking} OVN-Kubernetes network plugin allows the configuration of secondary network interfaces for pods. To configure secondary network interfaces, you must define the configurations in the `NetworkAttachmentDefinition` custom resource definition (CRD).
+The {openshift-networking} OVN-Kubernetes network plugin allows the configuration of secondary network interfaces for pods. To configure secondary network interfaces, you must define the configurations in the `NetworkAttachmentDefinition` custom resource (CR).
 
-:FeatureName: Configuration for an OVN-Kubernetes additional network
-include::snippets/technology-preview.adoc[]
+You can configure an OVN-Kubernetes additional network in either *layer 2* or *localnet* topologies.
+
+- A layer two topology supports east-west cluster traffic, but does not allow access to the underlying physical network.
```

Comment:
Should this still use the numerical format since its references numerically in the previous statement